app/src/components/Shared/ContentAccordion.jsx

```diff
@@ -51,7 +51,7 @@ class ContentAccordion extends Component {
     return (
       <Accordion
         allowMultiple={false}
-        activeItems={[-1]}
+        activeItems={0}
```

Comment:
do we want to have the first open by default? feels a bit weird to me tbh but please check with Dani